boards/arm/cc3235sf_launchxl/support/openocd.cfg

```diff
@@ -0,0 +1,10 @@
+#
+# TI CC3235SF-LaunchXL Evaluation Kit
+#
+source [find interface/xds110.cfg]
+transport select swd
+source [find target/ti_cc3220sf_vr.cfg]
+if { ![using_hla] } {
```

Comment:
Was this bit taken from previous boards/arm/cc3220sf_launchxl/support/openocd.cfg?  If so, it was a workaround for the  cc3220sf reset issues, and may not apply for the cc3235sf_launchxl (unless that board has the same reset/debug issue). 
Could you check against the cfg file from the TI OpenOCD release for cc3235sf, and probably use that one instead?

Reply:
@GAnthony TI OpenOCD does not have any cfg file specific to the CC3235SF. So I have been using the same files as CC3220SF. If I revert to using openocd/tcl/board/ti_cc3220sf_launchpad.cfg, without your workaround, I see the following failure when running 'make flash':

> Using runner: openocd
> Open On-Chip Debugger 0.10.0
> Licensed under GNU GPL v2
> For bug reports, read
> 	http://openocd.org/doc/doxygen/bugs.html
> adapter speed: 2500 kHz
> srst_only separate srst_gates_jtag srst_open_drain connect_deassert_srst
> adapter_nsrst_delay: 1100
> Info : XDS110: connected
> Info : XDS110: firmware version = 2.3.0.17
> Info : XDS110: hardware version = 0x0027
> Info : XDS110: connected to target via SWD
> Info : XDS110: SWCLK set to 2500 kHz
> Info : clock speed 2500 kHz
> Info : SWD DPIDR 0x2ba01477
> Info : cc32xx.cpu: hardware has 6 breakpoints, 4 watchpoints
>     TargetName         Type       Endian TapName            State       
> --  ------------------ ---------- ------ ------------------ ------------
>  0* cc32xx.cpu         cortex_m   little cc32xx.dap         running
> Error: DP initialisation failed
> in procedure 'reset' 
> in procedure 'ocd_bouncer'
> 
> 
> Info : XDS110: disconnected

Reply:
OK, thanks for checking.